pkg/verifier/simple/policy.go

```diff
@@ -62,7 +62,6 @@ func (v verifier) TransformPolicies(kWrapper kubernetes.WrapperInterface, namesp
 			if err != nil {
 				return nil, err
 			}
-			break
```

Comment:
Looks like a good catch. Was this previously breaking out of the surrounding `for` loop (which was presumably undesirable)?